Framework/src/EventFile.cxx

```diff
@@ -126,7 +132,9 @@ namespace ldmx {
             if (!parent_->tree_) {
                 EXCEPTION_RAISE("EventFile", "No event tree in the file");
             }
-            tree_ = parent_->tree_->CloneTree(0);
+	    if ( isOutputFile_ && !tree_) {
```

Comment:
This part will conflict with PR #566 , but keeping the other PR's version of these lines will allow both to work.